pkg/model/cluster_test.go

```diff
@@ -138,30 +138,30 @@ func testCluster(t *testing.T, describe spec.G, it spec.S) {
 			rawSubject.replicasActive.Add(secondReplica)
 
 			subject.RecordToAutoscaler(autoscalerFake, &theTime)
-			bufferRecorded = autoscalerFake.recorded[0]
+			routingStockRecorded = autoscalerFake.recorded[0]
 		})
 
-		// TODO immediately record arrivals at buffer
+		// TODO immediately record arrivals at rotingStock
```

Comment:
Typo here: `rotingStock` -> `routingStock`

Reply:
Fixed in https://github.com/pivotal/skenario/pull/67/commits/b3c3d8657d2149597dd78d8f0701746ca53862f2 commit.